nautobot_golden_config/models.py

```diff
@@ -350,6 +425,28 @@ def save(self, *args, **kwargs):
         self.missing = compliance_details["missing"]
         self.extra = compliance_details["extra"]
 
+    def remediation_on_save(self):
+        """The actual remediation happens here, before saving the object."""
+        if self.compliance:
+            self.remediation = None
+            return
+
+        if not self.rule.config_remediation:
```

Comment:
This will have no effect , but technically this should be the first rule in order

Reply:
@mzbroch Can you expand on why you're saying here?